### PR TITLE
Add distribution_id into first session ping

### PR DESCRIPTION
**Pull request by rxumoz**
_distribution_id_ have already been added to metrics ping (#16075)
We also want to add this field to other pings for convenience and reference.
This PR tries to add distribution_id to the first session ping.

### CODE REVIEW

**Comment by rxumoz** [comment]
## Request for data collection review
1.	What questions will you answer with this data?
This data helps us distinguish in telemetry between builds from different distributors.
2.	Why does Mozilla need to answer these questions? Are there benefits for users? Do we need this information to address product or business requirements?
Yes, this gives us an insight on methods of distribution.
3.	What alternative methods did you consider to answer these questions? Why were they not sufficient?
No alternative is available to answer this question.
4.	Can current instrumentation answer these questions?
Yes, but adding this field helps us to verify the distribution_id of the metrics.
5.	List all proposed measurements and indicate the category of data collection for each measurement, using the Firefox data collection categories found on the Mozilla wiki.

Measurement Description | Data Collection Category | Tracking Bug 
-- | -- | --
Distribution ID | category 1 - technical data | #22543 

6.	Please provide a link to the documentation for this data collection which describes the ultimate data set in a public, complete, and accurate way.
https://dictionary.telemetry.mozilla.org/apps/fenix/pings/first-session
7.	How long will this data be collected? Choose one of the following:
Permanently if possible.
8.	What populations will you measure?
No filters - all locales, channels, etc.
9.	If this data collection is default on, what is the opt-out mechanism for users?
Default Glean SDK opt-out mechanism.
10.	Please provide a general description of how you will analyze this data.
To verify the distribution from metrics.
11.	Where do you intend to share the results of your analysis?
Internally in the fenix/android-components/data science teams.
12.	Is there a third-party tool (i.e. not Telemetry) that you are proposing to use for this data collection?
No third-party tool will use this data.

File: app/metrics.yaml [hunk 1]
```diff
@@ -4370,6 +4370,22 @@ first_session:
...
+    send_in_pings:
+      - first-session
+    bugs:
+      - https://github.com/mozilla-mobile/fenix/issues/20376
+    data_reviews:
+      - https://github.com/mozilla-mobile/fenix/pull/22543#issuecomment-977456848
+    data_sensitivity:
+      - technical
+    notification_emails:
+      - android-probes@mozilla.com
```
Comment (rocketsroger) [inline app/metrics.yaml:4387]: When adding a telemetry permanently, please add your email to the `notification_emails` as the IC that is responsible for collecting this data.

**Review by rocketsroger — CHANGES_REQUESTED** [review]
Looks good.  One small issue for adding this telemetry permanently.

File: app/metrics.yaml (hunk 1 above)
Comment (rxumoz) [reply app/metrics.yaml:4387]: 👌thx

**Comment by rocketsroger** [comment]
> ## Request for data collection review
> 
>     1. What questions will you answer with this data?
>        This data helps us distinguish in telemetry between builds from different distributors.
> 
>     2. Why does Mozilla need to answer these questions? Are there benefits for users? Do we need this information to address product or business requirements?
>        Yes, this gives us an insight on methods of distribution.
> 
>     3. What alternative methods did you consider to answer these questions? Why were they not sufficient?
>        No alternative is available to answer this question.
> 
>     4. Can current instrumentation answer these questions?
>        Yes, but adding this field helps us to verify the distribution_id of the metrics ping.
> 
>     5. List all proposed measurements and indicate the category of data collection for each measurement, using the Firefox data collection categories found on the Mozilla wiki.
> 
> 
> Measurement Description 	Data Collection Category 	Tracking Bug
> Distribution ID 	category 1 - technical data 	#22543
> 
>     6. Please provide a link to the documentation for this data collection which describes the ultimate data set in a public, complete, and accurate way.
>        https://dictionary.telemetry.mozilla.org/apps/fenix/pings/first-session
> 
>     7. How long will this data be collected? Choose one of the following:
>        Permanently if possible.
> 
>     8. What populations will you measure?
>        No filters - all locales, channels, etc.
> 
>     9. If this data collection is default on, what is the opt-out mechanism for users?
>        Default Glean SDK opt-out mechanism.
> 
>     10. Please provide a general description of how you will analyze this data.
>         To verify the distribution from metrics.
> 
>     11. Where do you intend to share the results of your analysis?
>         Internally in the fenix/android-components/data science teams.
> 
>     12. Is there a third-party tool (i.e. not Telemetry) that you are proposing to use for this data collection?
>         No third-party tool will use this data.

# Data Review

1) Is there or will there be **documentation** that describes the schema for the ultimate data set in a public, complete, and accurate way? 

Yes, through the metrics.yaml file and the [Glean Dictionary](https://dictionary.telemetry.mozilla.org/apps/fenix )

2) Is there a control mechanism that allows the user to turn the data collection on and off? 

Yes, through the "Send Usage Data" preference in the application settings

3) If the request is for permanent data collection, is there someone who will monitor the data over time?

rxu@mozilla.com will be the IC responsible for monitoring this data over time.

4) Using the **[category system of data types](https://wiki.mozilla.org/Data_Collection)** on the Mozilla wiki, what collection type of data do the requested measurements fall under?

Category 1, technical data

5) Is the data collection request for default-on or default-off?

default-on

6) Does the instrumentation include the addition of **any *new* identifiers**?

No

7) Is the data collection covered by the existing Firefox privacy notice? 

Yes

8) Does the data collection use a third-party collection tool? 

No

## Result

data-review+

**Review by rocketsroger — APPROVED** [review]
Thanks 🚢

**Pull request merged**